samples: wifi: sta: Fix ground connections (#9941)
nrfconnect/sdk-nrf: samples/wifi/sta/README.rst

Hardware modifications
======================

> To measure the current consumption of the nRF5340 SoC in nRF7002 DK, complete the following steps:

Reviewer: ```suggestion
To measure the current consumption of the nRF5340 SoC in the nRF7002 DK, complete the following steps:
```

 * Make a short on solder bridge **SB17**.
 * Connect **GND** on PPK2 kit to **GND** on the nRF7002 DK.

   You can use **P4** pin **7** mentioned as **GND** for ground.
   Note that this connection requires a berg pin connector.

 * Connect the **Vout** on PPK2 to **P22** pin **1** on the nRF7002 DK.

To measure the current consumption of the nRF7002 device in nRF7002 DK, complete the following steps:

 * Remove jumper on **P23** (VBAT jumper).
 * Connect **GND** on PPK2 kit to **GND** on the nRF7002 DK.

   You can use **P21** pin **1** mentioned as **-** (**MINUS**) for ground.

 * Connect the **Vout** on PPK2 to **P23** pin **1** on the nRF7002 DK.

The following diagram illustrates a typical configuration for measuring current on the nRF7002 DK for both the nRF5340 SoC and the nRF7002 device:

   .. figure:: ../../../doc/nrf/images/nrf7002_nrf5340_current_measurements.svg
      :alt: Typical configuration for measuring current on the nRF7002 DK for both the nRF5340 SoC and the nRF7002 device.

      Typical configuration for measuring current on the nRF7002 DK for both the nRF5340 SoC and the nRF7002 device.

PPK2 usage and measurement
==========================

> To measure the current consumption of the nRF5340 SoC and nRF7002 device, complete the following steps:

Reviewer: ```suggestion
To measure the current consumption of the nRF5340 SoC and the nRF7002 device, complete the following steps:
```

> * Configure PPK2 connected to nRF5340 SoC as a source meter with 1.8 volts.
> * Configure PPK2 connected to nRF7002 device as a source meter with 3.6 volts.

Reviewer: ```suggestion
* Configure PPK2 connected to the nRF5340 SoC as a source meter with 1.8 volts.
* Configure PPK2 connected to the nRF7002 device as a source meter with 3.6 volts.
```

 * Remove jumper on **P22** (VDD jumper).
 * Cut solder bridge **SB16**.
 * Make a short on solder bridge **SB17**.

>  * Connect **GND** on PPK2 kit to **GND** on the nRF5340 SoC.

Reviewer 2: Replace nRF5340 with nRF7002 DK

Author: No, this was intentional, DK covers both 5340 SoC + 7002 device, so, to clarify we are using individual component names.

     You can use **P4** pin **7** mentioned as **GND** for ground.

     Note that this connection requires a berg pin connector.

>  * Connect the **Vout** on PPK2 to **P22** pin **1** on the nRF5340 SoC.

Reviewer 2: Replace nRF5340 with nRF7002 DK

Author: No, this was intentional, DK covers both 5340 SoC + 7002 device, so, to clarify we are using individual component names.

Author: Sorry, after re-reading, I understand your intention, will fix.

To measure the current consumption of the nRF7002 device, complete the following steps:

 * Remove jumper on **P23** (VBAT jumper).

>  * Connect **GND** on PPK2 kit to **GND** on the nRF7002 device.

Reviewer 2: Replace nRF7002 device with nRF7002 DK

     You can use **P21** pin **1** mentioned as **-** for ground.

>  * Connect the **Vout** on PPK2 to **P23** pin **1** on the nRF7002 device.

Reviewer 2: Replace nRF7002 device with nRF7002 DK

 * Cut solder bridge **SB16**.
 * Make a short on solder bridge **SB17**.
 * Connect **GND** on PPK2 kit to **GND** on the nRF7002 DK.

>     You can use **P4** pin **7** mentioned as **GND** for ground.

Reviewer: ```suggestion
   You can use **P4** pin **7** mentioned as **GND** for ground.
```

Reviewer: The formatting needs to be corrected as it makes lines 182 and 190 **bold**.

Reviewer: No. The lines which are appearing in bold needs to be in regular font.

>     Please note this needs a berg pin connector.

Reviewer: ```suggestion
    Note that this connection requires a berg pin connector.
```

 * Connect the **Vout** on PPK2 to **P22** pin **1** on the nRF7002 DK.

To measure the current consumption of the nRF7002 device, complete the following steps:

 * Remove jumper on **P23** (VBAT jumper).
 * Connect **GND** on PPK2 kit to **GND** on the nRF7002 DK.

>     You can use **P21** pin **1** mentioned as **-** for ground.

Reviewer: ```suggestion
   You can use **P21** pin **1** mentioned as **-** for ground.
```

 * Remove jumper on **P23** (VBAT jumper).
 * Connect **GND** on PPK2 kit to **GND** on the nRF7002 DK.

>     You can use **P4** pin **7** mentioned as  **GND** (-).

Reviewer 2: Remove (-) in this the line. 
In line number 183 please make as **-** for GND
